Arrowgene.Ddon.GameServer/Handler/LobbyLobbyDataMsgHandler.cs

```diff
@@ -53,9 +54,12 @@ public override void Handle(GameClient client, StructurePacket<C2SLobbyLobbyData
             else
             {
                 // We are in one of the common areas where players can see eachother
+		        List<GameClient> otherClients = Server.ClientLookup.GetAll()
+                                                    .Where(x => x != null)
+                                                    .ToList();
```

Comment:
You still got the LINQ on top of the foreach, now that the foreach does a null check it should be removed

Reply:
🤦‍♂️ fixed